synapse/storage/data_stores/main/state.py

```diff
@@ -246,14 +246,10 @@ def __init__(self, db_conn, hs):
         # We size the non-members cache to be smaller than the members cache as the
         # vast majority of state in Matrix (today) is member events.
 
-        self._state_group_cache = DictionaryCache(
-            "*stateGroupCache*",
-            # TODO: this hasn't been tuned yet
-            50000 * get_cache_factor_for("stateGroupCache"),
-        )
+        # TODO: this hasn't been tuned yet
+        self._state_group_cache = DictionaryCache("*stateGroupCache*", 50000)
         self._state_group_members_cache = DictionaryCache(
-            "*stateGroupMembersCache*",
-            500000 * get_cache_factor_for("stateGroupMembersCache"),
+            "*stateGroupMembersCache*", 500000
```

Comment:
Why does this no longer have a cache factor?

Reply:
because now DictionaryCache does the cache factor math, not the things that call it

Reply:
Sorry, I'm not following where its doing that? AFAICT `get_cache_factor_for` is only called for a single cache?

Reply:
right, but now you can configure any of the cache sizes, so it's done in the caches itself